ambry-server/src/main/java/com.github.ambry.server/StatsManager.java

```diff
@@ -134,7 +135,7 @@ void collectAndAggregate(StatsSnapshot aggregatedSnapshot, PartitionId partition
       try {
         long fetchAndAggregatePerStoreStartTimeMs = time.milliseconds();
         StatsSnapshot statsSnapshot = store.getStoreStats().getStatsSnapshot();
-        aggregate(aggregatedSnapshot, statsSnapshot);
+        StatsSnapshot.aggregate(aggregatedSnapshot, statsSnapshot);
```

Comment:
At the moment all changes are made to keep the old features functional. If you would like too, i can also just remove it.